src/MyMarketManager.Data/Entities/ProductPhoto.cs

```diff
@@ -12,5 +12,30 @@ public class ProductPhoto : EntityBase
 
     [Required]
     public string Url { get; set; } = string.Empty;
+    
+    /// <summary>
+    /// User-provided or AI-generated caption describing the image.
+    /// </summary>
     public string? Caption { get; set; }
+    
+    /// <summary>
+    /// AI-generated description of the image content.
+    /// </summary>
+    public string? AiDescription { get; set; }
```

Comment:
These look to be the same. Are they? What is the difference or should we delete one?